src/mousetracker.js

```diff
@@ -3248,5 +3264,33 @@
             } );
         }
     }
+    
+    /**
+     * @function
+     * @private
+     * @inner
+     * @returns {Boolean} True if inside an iframe, otherwise false.
+     */
+    function isInIframe () {
+        try {
+            return window.self !== window.top;
+        } catch (e) {
+            return true;
+        }
+    }
```

Comment:
Sure, updated the pull request with your code. Tested it and it seems to work the same.